pkg/gadget-service/service-oci.go

```diff
@@ -122,6 +131,11 @@ func (s *Service) RunGadget(runGadget api.GadgetManager_RunGadgetServer) error {
 		return fmt.Errorf("expected first control message to be gadget run request")
 	}
 
+	metricAttribs := attribute.NewSet(
+		attribute.KeyValue{Key: "gadget-image", Value: attribute.StringValue(ociRequest.ImageName)},
```

Comment:
I see it's converted to `gadget_image` on Pormetheus. Should we rename it to avoid any confusion later on? (same in other places)